docs/zh/latest/admin-api.md

```diff
@@ -665,11 +668,42 @@ HTTP/1.1 200 OK
 
 ```
 
+示例二：如何代理上游 https。
+
+1、创建 route 并 配置 upstream 的 scheme 为 `https`。
+
+```shell
+$ curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uri": "/get",
+    "upstream": {
+        "type": "roundrobin",
+        "scheme": "https",
+        "nodes": {
+            "httpbin.org:443": 1
+        }
+    }
+}'
+```
+
+执行成功后，请求与上游通信时的 scheme 将为 `https`。
+
+2、 发送请求进行测试。
+
+```shell
+$ curl http://127.0.0.1:9080/get -i
```

Comment:
IMHO, this test case lacks persuasion. You'd better also print the result returned by the upstream.